functions/copy_creds.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+kubectl cp ~/.aws benchmark-pod-1:/root/.aws -c benchmark-1
+kubectl cp ~/.aws benchmark-pod-1:/root/.aws -c benchmark-2
+kubectl cp ~/.aws benchmark-pod-1:/root/.aws -c benchmark-3
+kubectl cp ~/.aws benchmark-pod-1:/root/.aws -c benchmark-4
+kubectl cp ~/.aws benchmark-pod-2:/root/.aws -c benchmark-4
+kubectl cp ~/.aws benchmark-pod-2:/root/.aws -c benchmark-3
+kubectl cp ~/.aws benchmark-pod-2:/root/.aws -c benchmark-2
+kubectl cp ~/.aws benchmark-pod-2:/root/.aws -c benchmark-1
+kubectl cp ~/.aws benchmark-pod-3:/root/.aws -c benchmark-1
+kubectl cp ~/.aws benchmark-pod-3:/root/.aws -c benchmark-2
+kubectl cp ~/.aws benchmark-pod-3:/root/.aws -c benchmark-3
+kubectl cp ~/.aws benchmark-pod-3:/root/.aws -c benchmark-4
```

Comment:
what are these calls for?
what does the numbers (1,2,3,4) mean? number of threads?